readme.md

```diff
@@ -29,8 +29,14 @@ sparkly([0, 3, 5, '', 4, 3, 4, 10]);
 // specifying a min max object will change the sparkline range
 sparkly([1, 2, 3, 4, 5], {min: 0, max: 10});
 //=> '▁▂▃▄▄'
+
+// specifying a style object will change the sparkline color
+sparkly([1, 2, 3, 4, 5], {style: 'fire'});
+//=>'▁▂▃▄▅▆▇█'
 ```
 
+colored sparkline will look like this: ![sparkly](http://i.imgur.com/5SUQ4mz.png)
```

Comment:
Add the image directly to the repository or like the logo, add to `cloud.githubusercontent`. Not sure how do that cloud stuff though.